cmd/skupper/skupper_token.go

```diff
@@ -44,3 +51,66 @@ func NewCmdTokenCreate(skupperClient SkupperTokenClient, flag string) *cobra.Com
 	skupperClient.CreateFlags(cmd)
 	return cmd
 }
+
+var TokenStatusOpts types.TokenStatus
+
+func NewCmdTokenStatus(skupperClient SkupperTokenClient, flag string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:    "status <output-token-file>",
+		Short:  "display token information",
```

Comment:
Not a blocker: the description could follow the style of other commands' descriptions, perhaps providing a glimpse of the type of information that is being shown.

Reply:
Sorry for the confusion, I was referring to change the property `Short` of the cobra.Command struct (line 60), that is going to be shown with the --help flag. 

This could be an example, but feel free to modify it: 
```
Short:  "Displays information about the token, including its metadata, expiration data, and claim details."
```

Thank you!

Reply:
Thank you for the clarification.   I have added the Short command description.